pipeline_tools/create_analysis_json.py

```diff
@@ -2,22 +2,42 @@
 
 import json
 import argparse
-import requests
 
 
 def create_analysis(analysis_id, metadata_file, input_bundles_string, reference_bundle,
         run_type, method, schema_version, inputs_file, outputs_file, format_map):
+    """Creates analysis json for submission.
+
+    Args:
+        analysis_id: String representing the workflow id of the analysis
+        metadata_file: String representing path to file containing metadata json for the workflow
+        input_bundles_string: String containing a comma-separated list of input bundle uuids
+        reference_bundle: String representing the uuid of the reference bundle
+        run_type: String indicating whether this analysis was actually run or is just copying previous results
+        method: String representing the name of the workflow
+        schema_version: String giving the version of the metadata schema that the analysis json conforms to
+        inputs_file: String representing path to file containing metadata about workflow inputs
+        outputs_file: String representing path to file containing metadata about workflow outputs
+        format_map: String representing path to file containing a map of file extensions to types
+
+    Returns:
+        Dictionary representing the analysis json file to be submitted
+    """
     print('Creating analysis.json for {}'.format(analysis_id))
 
     with open(metadata_file) as f:
         metadata = json.load(f)
-        start, end = get_start_end(metadata)
+        start = metadata['start']
+        end = metadata['end']
+        print(start, end)
         tasks = get_tasks(metadata)
 
     inputs = create_inputs(inputs_file)
     outputs = create_outputs(outputs_file, format_map, schema_version)
 
-    input_bundles = get_input_bundles(input_bundles_string)
+    input_bundles = input_bundles_string.split(',')
+    print(input_bundles)
```

Comment:
Would be better to have some information around this:, e.g. `print('Input Bundles: {}.format(input_bundles)')`